### PR TITLE
[12.0][ADD]account_tax_description_translatable

### PR DESCRIPTION
**Pull request by Ryoko04**
This pull request adds the function for "Tax description should be translatable #804".

### CODE REVIEW

File: account_tax_description_translatable/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+* `Quartile Limited <https://www.quartile.co>`__:
+
+  * Yoshi Tashiro <tashiro@quartile.co>
+  * Tim Lai <tl@quartile.com>
```
Comment (yostashiro) [inline account_tax_description_translatable/readme/CONTRIBUTORS.rst]:
```suggestion
```
Just your name will do.

File: account_tax_description_translatable/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+This module allows to translate the tax description.
```
Comment (yostashiro) [inline account_tax_description_translatable/readme/DESCRIPTION.rst]:
```suggestion
This module allows to translate the tax description to show in the printed quotation/purchase order/invoice.
```

**Comment by legalsylvain** [comment]
Hi. @Ryoko04 

This module seems a fix. Don't you think a PR against odoo or OCB could be better ?

**Comment by pedrobaeza** [comment]
AFAIK, you get this installing core module `l10n_multilang`.

**Pull request closed without merging**